addons/binding/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/internal/hardware/FritzahaWebInterface.java

```diff
@@ -91,7 +91,7 @@
     public String authenticate() {
         if (this.config.getPassword() == null) {
             this.fbHandler.setStatusInfo(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR,
-                    "please configure password first");
+                    "no password set");
```

Comment:
What was wrong with the old text?

Reply:
To be honest: I really can't remember why I changed it. Will revert the change.